cartography/intel/gcp/storage.py

```diff
@@ -0,0 +1,204 @@
+import logging
+
+from googleapiclient.discovery import HttpError
+
+from cartography.intel.gcp import compute
+from cartography.util import run_cleanup_job
+logger = logging.getLogger(__name__)
+
+
+def get_gcp_buckets(storage, project_id):
+    """
+    Returns a list of storage objects within some given project
+
+    :type storage: The GCP storage resource object
+    :param storage: The storage resource object created by googleapiclient.discovery.build()
+
+    :type project_id: str
+    :param project_id: The Google Project Id that you are retrieving buckets from
+
+    :rtype: Storage Object
+    :return: Storage response object
+    """
+    try:
+        req = storage.buckets().list(project=project_id)
+        res = req.execute()
+        return res
+    except HttpError as e:
+        reason = compute._get_error_reason(e)
+        if reason == 'invalid':
+            logger.debug(
+                (
+                    "The project %s is invalid - returned a 400 invalid error."
+                    "Full details: %s"
+                ),
+                project_id,
+                e,
+            )
+            return None
+        elif reason == 'forbidden':
+            logger.debug(
+                (
+                    "You do not have storage.bucket.list access to the project %s. "
+                    "Full details: %s"
+                ), project_id, e, )
+            return None
```

Comment:
As above, can you please change the `logger.debug` to `logger.warning`?

Same with `return {}`.

Reply:
Done!